bin/ytdl.js

```diff
@@ -296,6 +297,14 @@ if (opts.infoJson) {
       if (!ext && format.container) {
         output += '.' + format.container;
       }
+
+      // Parses & sanitises output filename for any illegal charcters
+      var parsedOutput = path.posix.parse(output);
+      output = path.format({
```

Comment:
just noticed, I don't get it at all, why do we need `path.parse()` and `path.format()`?
I replaced the whole commit with a single line `output = sanitizeName(output)` and it works well as expected(at least for me, linux+win)

Reply:
In case the user wants to specify the directory of the output file. If you do that with `output = sanitizeName(output)`, the path separators (/ or \) will be removed by `sanitize-filename`